Application/global_settings_manager.py

```diff
@@ -86,3 +97,53 @@ def load_global_settings(self):
 
         settings.endGroup()
         settings.endGroup()
+        settings.endGroup()
+
+    def set_to_user_settings(self):
+        """
+        Saves the values in the global settings entity to Qsettings
+        """
+        settings = QSettings()
+
+        settings.beginGroup("global-settings")
+
+        settings.beginGroup("user-settings")  # open user settings
+
+        # Sets the padding and the max width.
+        settings.setValue("table_padding", self.global_settings_entity.table_padding)
+        settings.setValue("table_maximum_width", self.global_settings_entity.table_maximum_width)
+
+        # Sets the cell size with width, index 0, and height, index 1.
+        settings.setValue("table_cell_size_width", self.global_settings_entity.table_cell_size[0])
+        settings.setValue("table_cell_size_height", self.global_settings_entity.table_cell_size[1])
+
+        settings.endGroup()
+        settings.endGroup()
+
+    def set_table_padding(self, table_padding):
+        """
+        Setter method to set the cell padding.
```

Comment:
Add where we're setting the data to. These comments are useful because we can look at documentation of a method from another file, so the more context provided the more convenient the documentation will be.